### DISCUSSION TITLE
How can I display an image in my chatbox?

### DISCUSSION THREAD

**Discussion by thon16** [longy2k/obsidian-bmo-chatbot#68] [Q&A]
Hi, 

I notice there is an image displayed in the chatbox, and the model is mistral-medium. 
[https://github.com/longy2k/obsidian-bmo-chatbot/blob/main/README_images/eli5-example.png?raw=true](url)
![image](https://github.com/longy2k/obsidian-bmo-chatbot/assets/18422861/5bd78f81-c22e-477c-9ad9-0777bfc80bf6)

I tried to reproduce the result, but mistral is a text-based LLM, It doesn't have capability to generate images directly.
I also tried to ask for a http link which contains a picture of Subway Surfers, it also didn't work.

So, the question is, how can I have this feature work properly?

**Comment by twalderman** [comment]
If the model sends an HTML link for an image, an image will appear.  It works well when and if the model has a link in its context that is valid.
